packages/core/src/components/DismissableBanner/DismissableBanner.stories.tsx

```diff
@@ -100,3 +100,15 @@ export const WithLink = () => (
     </ApiProvider>
   </div>
 );
+export const Fixed = () => (
+  <div style={containerStyle}>
+    <ApiProvider apis={apis}>
+      <DismissableBanner
+        message="This is a dismissable banner with a fixed position fixed at the bottom of the page"
+        variant="info"
+        id="fixed_dismissable"
```

Comment:
Can we drop the id here?

Reply:
hm they all have one, i wonder why

Reply:
So you mean I remove the id?

Reply:
Yeah, that's what I meant, but saw your comment above, let's leave it intact then!